components/script/dom/window.rs

```diff
@@ -659,6 +660,14 @@ impl WindowMethods for Window {
     }
 
     #[allow(unsafe_code)]
+    fn CrashHard(&self) {
+        let p: *mut u32 = ptr::null_mut();
```

Comment:
Did you sanity check that the optimizer doesn't do anything weird here? I don't know what LLVM does but in Breakpad's unit tests we had to jump through hoops to avoid GCC's optimizer optimizing away our crash function:
https://chromium.googlesource.com/breakpad/breakpad/+blame/master/src/client/linux/handler/exception_handler_unittest.cc#195